test/functional/tensor/matrix/test-tensor-matrix-double.hpp

```diff
@@ -14,58 +14,58 @@ using MatrixElementType = double;
 using TensorMatrixTypes = ::testing::Types<
 
 #ifdef RAJA_ENABLE_CUDA
-    RAJA::RectMatrixRegister<MatrixElementType, TensorMatrixLayoutType, 8,4, RAJA::cuda_warp_register>,
-    RAJA::RectMatrixRegister<MatrixElementType, TensorMatrixLayoutType, 8,8, RAJA::cuda_warp_register>,
+    RAJA::expt::RectMatrixRegister<MatrixElementType, TensorMatrixLayoutType, 8,4, RAJA::expt::cuda_warp_register>,
```

Comment:
That's odd, I thought these were already under `::expt`?

Reply:
We had an alias to the expt namespace that allowed you to call anything under it without expt explicitly. I had to remove that as it broken MSVC builds.